_data/hipaa-landing-page-tabs-ref-arch-code.yml

```diff
@@ -0,0 +1,167 @@
+- id: infra-terraform
+  title: Terraform
+  content: |
+    <figure>
+      <pre>
+      <code class="language-hcl">
+        ########################################
+        # EKS Cluster Configuration in Terraform
+        ########################################
+
+        # ---------------------------------------------------------------------------------------------------------------------
+        # CONFIGURE OUR KUBERNETES CONNECTIONS
+        # Note that we can't configure our Kubernetes connection until EKS is up and running, so we try to depend on the
+        # resource being created.
+        # ---------------------------------------------------------------------------------------------------------------------
+
+        # The provider needs to depend on the cluster being setup.
+        provider "kubernetes" {
```

Comment:
This code example doesn't show users what they probably look for. What is a `provider` block? How does this communicate that our code is in versioned modules? That we have out-of-the-box solutions for lots of EKS complexity?

See the [EKS example (click the tab) on the Heroku page](https://gruntwork.io/migrate-from-heroku-to-aws#step-4-deploy-to-aws) for a slightly clearer example you can use.

Reply:
Swapped in the EKS example from the Heroku page in d31f1d14